examples/offline_inference/mlpspeculator.py

```diff
@@ -7,7 +14,7 @@
 
 
 def time_generation(llm: LLM, prompts: list[str],
-                    sampling_params: SamplingParams):
+                    sampling_params: SamplingParams, print_str: str):
```

Comment:
```suggestion
                    sampling_params: SamplingParams, title: str):
```

Suggest to rename this to `title`